docs/grid/custom_filter.md

```diff
@@ -1,7 +1,7 @@
-Creating custom filter
+Creating custom filters
```

Comment:
Not sure about this one. While it sounds more natural, do we need to rename file and change entry at table of contents at `index.md`? And it should be synced with other files like `custom_action`, `custom field type`, `custom bulk action`

WDYT @loic425

Reply:
Fair point (I forgot to change the links in grid/index.md !). In my opinion, they should all be plural or they could all have an article (so 'creating **a** custom filter' / 'creating **a** custom action' ...) because it sounds more natural, even though it is correct without, but it's up to you guys :)